views/add_user.jade

```diff
@@ -0,0 +1,25 @@
+extends layout
+
+block content
+    h1 #{title}
```

Comment:
Add user is done by sign up right? Instead of the admin adding.

Reply:
In most circumstances, users will be added via sign up. But admin should also have the capability to add users?

Reply:
yeah possible